Dockerfile_db

```diff
@@ -0,0 +1,5 @@
+#
+FROM kartoza/postgis:14-3.2
+
+# install postgis commandline tools such as shp2pgsql
+#RUN apt-get update && apt-get install -y postgis
```

Comment:
Waarom is dit weggecomment? Als het niet gebruikt wordt dan moet het gewoon weg.